dotnet/WipeAppWriteBucket/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Net;
+using Appwrite;
+using Newtonsoft.Json;
+using WipeAppWriteBucket.Models;
+
+namespace WipeAppWriteBucket
+{
+    class Program
```

Comment:
The function structure is incorrect and should be as shown in the [.NET Example here in our docs](https://appwrite.io/docs/functions#writingYourOwnFunction)